Makefile

```diff
@@ -4,67 +4,86 @@ TARGET_NAME_XCFRAMEWORK = $(EXTENSION_NAME).xcframework
 SCHEME_NAME_XCFRAMEWORK = AEPCampaignClassic
 CURR_DIR := ${CURDIR}
 
-SIMULATOR_ARCHIVE_PATH = ./build/ios_simulator.xcarchive/Products/Library/Frameworks/
+SIMULATOR_ARCHIVE_PATH = $(CURR_DIR)/build/ios_simulator.xcarchive/Products/Library/Frameworks/
 SIMULATOR_ARCHIVE_DSYM_PATH = $(CURR_DIR)/build/ios_simulator.xcarchive/dSYMs/
-IOS_ARCHIVE_PATH = ./build/ios.xcarchive/Products/Library/Frameworks/
+IOS_ARCHIVE_PATH = $(CURR_DIR)/build/ios.xcarchive/Products/Library/Frameworks/
 IOS_ARCHIVE_DSYM_PATH = $(CURR_DIR)/build/ios.xcarchive/dSYMs/
+IOS_DESTINATION = 'platform=iOS Simulator,name=iPhone 14'
 
 lint-autocorrect:
 	./Pods/SwiftLint/swiftlint autocorrect --format
 
 lint:
 	./Pods/SwiftLint/swiftlint lint
 
-setup:
-	(pod install)
-
-setup-tools: install-githook
-
-pod-repo-update:
-	(pod repo update)
+check-format:
+	swiftformat --lint AEPCampaignClassic/Sources --swiftversion 5.1
+	
+format:
+	swiftformat AEPCampaignClassic/Sources --swiftversion 5.1
 
-# pod repo update may fail if there is no repo (issue fixed in v1.8.4). Use pod install --repo-update instead
 pod-install:
 	(pod install --repo-update)
 
 ci-pod-install:
 	(bundle exec pod install --repo-update)
 
+pod-repo-update:
+	(pod repo update)
+
+install-swiftformat:
+	HOMEBREW_NO_AUTO_UPDATE=1 brew install swiftformat && brew cleanup swiftformat 
+
 pod-update: pod-repo-update
 	(pod update)
 
 open:
 	open $(PROJECT_NAME).xcworkspace
+	
+clean:
+	rm -rf build
 
-test:
+test: clean
 	@echo "######################################################################"
-	@echo "### Unit Testing iOS"
+	@echo "### Testing iOS"
 	@echo "######################################################################"
-	xcodebuild test -workspace $(PROJECT_NAME).xcworkspace -scheme $(PROJECT_NAME) -destination 'platform=iOS Simulator,name=iPhone 8' -derivedDataPath build/out -enableCodeCoverage YES
+	xcodebuild test -workspace $(PROJECT_NAME).xcworkspace -scheme $(PROJECT_NAME) -destination $(IOS_DESTINATION) -derivedDataPath build/out -resultBundlePath build/$(PROJECT_NAME).xcresult -enableCodeCoverage YES
 
 archive:
+	clean pod-install build
```

Comment:
This will not work, the target dependencies needs to go on the same line. Let's fix it as shown below:
```
archive: clean pod-install build
```

Reply:
Also, uncheck the "based on dependency analysis" box in Swift lint action